lib/src/uri_template.dart

```diff
@@ -225,7 +225,7 @@ class UriTemplate {
       _exprRegex,
       onMatch: (match) {
         parts.add(match);
-        return null;
+        return '';
```

Comment:
was this a bug? what is the implication of the change here?

Reply:
See by other comment.